CONTRIBUTING.md

```diff
@@ -0,0 +1,44 @@
+
+Changes to the process document tend to fall into a few classes, and how the W3C handles these varies. Simple editorial changes may require little review, but more substantial changes will require a more substantial review and discussions by the W3C community. Please read the below, and get involved as explained. 
+
+If you have any questions, please [raise an issue](/w3c/w3process/issues).
+
+## The W3C Process Community Group
+The W3C run a [Process Community Group](https://www.w3.org/community/w3process/) which is an open group created to manage discussions on developing the [W3C Process Document](https://www.w3.org/2017/Process-20170301/). Some suggestions for changes to the [W3C Process Document](https://www.w3.org/2017/Process-20170301/) will need to be made within the [Process Community Group](https://www.w3.org/community/w3process/), please see information below as to whether you need to take your suggestions to the group or can just raise a `pull request` or `issue`.
+
+## Simple Editorial Changes
+For simple editorial changes, please feel free to raise a `pull request` or [raise an issue](/w3c/w3process/issues) on this repository. A simple editorial change can be things like spelling and grammar mistakes, broken links, and so on.
+
+Some changes may appear simple but may actually be more complicated than they appear. Don't worry - please continue to send your `pull request` or [raise an issue](/w3c/w3process/issues) and we can discuss. 
+
+## Changes that Affect the Way the W3C Operates but Do Not Change the Policy
+For changes that affect the way that the W3C operates but that don't change policy please do the following:
+
+* [Raise an issue](/w3c/w3process/issues) on this repo
+* Make sure the [Process Community Group](https://www.w3.org/community/w3process/) adds the issue to it's agenda. This will allow you to explain why you think this is an improvement to the operation of the W3C (it's probably not helpful to provide a Pull Request until you have consensus on the approach).
+
+The best way to ensure your issue gets added to the [Process Community Group](https://www.w3.org/community/w3process/) agenda is to add yourself to the group, please visit [the group homepage](https://www.w3.org/community/w3process/) and select the button "Join or Leave this Group".
+
+Another way to ensure the [Process Community Group](https://www.w3.org/community/w3process/) addressed your issue is to check pack on the [issues page](/w3c/w3process/issues) page and ask whether it has been addressed. It is more effective to join the group, but you have this option if you need it. 
+
+If the [Process Community Group](https://www.w3.org/community/w3process/) reaches consensus on your issue, you may either raise a `pull request` or wait for the Editor to make the required changes. Note: raising a `pull request` **will** be faster. See more details about `pull requests` below.
+
+## Changes that Affect the Way the W3C Operates and Change the Policy
```

Comment:
I'm not sure this section is quite right.

The CG proposes, and potentially reaches consensus on changes to policy. Those are then reviewed by the AB (on whose behalf the CG does this) and unless they disagree, are presented to the AC for formal review and adoption.

So there is no inherent need to make the proposals through those bodies - doing it here is the right place to start.

You just need to be aware that unless those groups are happy, your changes will end up on the floor...

Reply:
This doesn't seem to be addressed. Changes to policy should be proposed to the CG first, in general - _inter alia_ to help achieve transparency. That was the point of doing work in a CG.

Reply:
No, I don't agree. If you bring something to the CG that is a clear policy change, what we've done (and I do as chair) is bounce it straight to the AB. Once a policy is settled, doing the consequent changes to the process is indeed the process CG's baby. I suppose we could say that they can raise it here if they like and have the CG take it to the AB, but I think it's better to have the proponent realize this and be the advocate at the AB/AC.

Reply:
That's great if the proponent happens to be on the AB, but otherwise it seems odd to have a group who only does editorial work when the policy has been agreed - that is what I understood as the primary responsibility of an editor.

Reply:
I think we have learned that crafting a process to reflect an agreed policy change is more than editorial work. Getting the details right is tricky.

I don't mind saying that the CG will refer policy changes to the AB/AC, since sometimes anyway it's not always clear what's a 'operational' question and what's a 'policy one'.

Reply:
Yeah, my model is that people should bring their issues to the CG. If the CG decides it needs guidance from the AB or AC before it considers the idea, OK, it can get that. But telling people they **must** do that legwork themselves before making a proposal here seems an unreasonably high barrier.

Reply:
feel free to offer that as an alternative; as long as it's clear the CG doesn't make policy, and that policy changes can start at AB/AC/CG but if they start at the CG, they will be referred